packages/graphql/src/translate/translate-update.ts

```diff
@@ -33,14 +33,15 @@ import translateTopLevelMatch from "./translate-top-level-match";
 import { createConnectOrCreateAndParams } from "./connect-or-create/create-connect-or-create-and-params";
 import { upperFirst } from "../utils/upper-first";
 
-function translateUpdate({ node, context }: { node: Node; context: Context }): [string, any] {
+function translateUpdate({ node, context, args }: { node: Node; context: Context, args?: any }): [string, any] {
     const { resolveTree } = context;
-    const updateInput = resolveTree.args.update;
-    const connectInput = resolveTree.args.connect;
-    const disconnectInput = resolveTree.args.disconnect;
-    const createInput = resolveTree.args.create;
-    const deleteInput = resolveTree.args.delete;
-    const connectOrCreateInput = resolveTree.args.connectOrCreate;
+    const rootArgs = { ...resolveTree.args, ...args };
+    const updateInput = rootArgs.update;
+    const connectInput = rootArgs.connect;
+    const disconnectInput = rootArgs.disconnect;
+    const createInput = rootArgs.create;
+    const deleteInput = rootArgs.delete;
+    const connectOrCreateInput = rootArgs.connectOrCreate;
```

Comment:
These variables are the main reason tck tests seem to be failing. They get the correct types on `dev`, but I can't seem to figure out what those should be.